src/app/screens/Books/index.tsx

```diff
@@ -21,4 +25,8 @@ function Books({navigation}: any) {
   );
 }
 
-export default Books;
+const mapStateToProps = (state: any) => ({
```

Comment:
You are not using this prop, also use `useSelector` instead of `mapStateToProps`